redisson/src/main/java/org/redisson/RedissonScoredSortedSet.java

```diff
@@ -372,11 +405,27 @@ public Double getScore(V o) {
         return get(getScoreAsync(o));
     }
 
+    @Override
+    public Collection<Double> getAllScore(List<V> keys) {
+        return get(getAllScoreAsync(keys));
+    }
+
     @Override
     public RFuture<Double> getScoreAsync(V o) {
         return commandExecutor.readAsync(getName(), StringCodec.INSTANCE, RedisCommands.ZSCORE, getName(), encode(o));
     }
 
+    @Override
+    public RFuture<Collection<Double>> getAllScoreAsync(Collection<V> elements) {
+        return redisson.getScript(DoubleCodec.INSTANCE).evalAsync(RScript.Mode.READ_ONLY,
+                        "local r = {} " +
+                        "for i, v in ipairs(KEYS) do " +
```

Comment:
@mrniko hope it's ok to use keys here instead of iterating over argv, I had some issues with encoding.

Reply:
no, please don't use params as keys.

Reply:
updated.